mariadb-10.11.yaml

```diff
@@ -0,0 +1,219 @@
+package:
+  name: mariadb-10.11
+  # Latest LTS
+  version: 10.11.2 # When bumping to 10.6.13, the patch for CVE-2022-47015 can be removed.
```

Comment:
This comment confuses me. When would we bump 10.11.2 (down) to 10.6.13?

edit: oh it's just copied.

Reply:
D'oh! The comment worked too (I removed the patch), but I didn't remove the comment :)